Author: Adam Gandelman <adamg@ubuntu.com>
Date: Wed Dec 18 10:06:25 PST 2013
Subject: Reverts stable/havana version bumps to requirements.txt

Reverts various version bumps to dependencies in stable/havana that
were updated in global requirements to fix gating issues and synced
across projects.

Index: cinder/requirements.txt
===================================================================
--- cinder.orig/requirements.txt	2013-12-18 10:11:29.047267359 -0800
+++ cinder/requirements.txt	2013-12-18 10:14:33.963265684 -0800
@@ -2,10 +2,10 @@
 amqplib>=0.6.1
 anyjson>=0.3.3
 argparse
-Babel>=1.3
+Babel>=0.9.6
 eventlet>=0.13.0
 greenlet>=0.3.2
-iso8601>=0.1.8
+iso8601>=0.1.4
 kombu>=2.4.8
 lockfile>=0.8
 lxml>=2.3
